chapters/annotations.tex

```diff
@@ -72,6 +72,19 @@ \subsection{Class Description and Revision History}\label{annotation-info-revisi
 <a href="modelica:/MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
 
+\subsubsection{Style sheet}
+
+A top-level package may contain an annotation \lstinline!styleSheet="modelica:/Package/Resources/style.css"!, which provides a cascading stylesheet.
+The given stylesheet is used when displaying the \lstinline!info! and \lstinline!revisions! annotations found in the \lstinline!Documentation! annotations.
+The provided stylesheet shall not use type or universal selectors.
+\begin{nonnormative}
+It is recommended to use class and id selectors with a \lstinline!$NameOfLibrary$-! prefix to avoid collisions when the content is included in a larger context.
+\end{nonnormative}
+
+Vendors shall use a \lstinline!$NameOfVendor$-! prefix to style vendor generated HTML content surrounding the user provided documentation.
+If tools want to give users (of that tool) the possibility to override the tool-specific CSS they can document that.
+The prefix is used to avoid this happening by accident.
+
```

Comment:
(Attaching comment to line not already occupied with other comments.)

Shouldn't `styleSheet` be part of `Documentation`?

Reply:
I can understand that idea, but it made me realize that I made an error in merging one of the suggestions.
The point is that the style sheet is given for the entire library (having multiple style sheets in one package would create a mess) - whereas Documentation is for each class.
I will restore text for that.

Reply:
This has been addressed.  Resolving.